mcli/mcli-hf-eval.yaml

```diff
@@ -10,49 +10,281 @@ command: |
-  # model_name_or_path: EleutherAI/pythia-6.9b
-  # model_name_or_path: facebook/opt-6.7b
-  # model_name_or_path: cerebras/Cerebras-GPT-6.7B
-  # model_name_or_path: StabilityAI/stablelm-base-alpha-7b
-  # model_name_or_path: EleutherAI/gpt-neox-20b
-  # model_name_or_path: EleutherAI/pythia-12b
-  # model_name_or_path: facebook/opt-13b
-  # model_name_or_path: cerebras/Cerebras-GPT-13B
-
-  # Tokenizer
-  tokenizer:
-    name: ${model_name_or_path}
-    kwargs:
-      model_max_length: ${max_seq_len}
-
-  model:
-    name: hf_causal_lm
-    pretrained_model_name_or_path: ${model_name_or_path}
-    init_device: cpu
-    pretrained: true
-    use_auth_token: false
+  device_eval_batch_size: 4
+  precision: amp_fp16
+
+  models:
+  -
+    model_name: mosaicml/mpt-7b-instruct
+    # Tokenizer
+    tokenizer:
+      name: EleutherAI/gpt-neox-20b
+      kwargs:
+        model_max_length: ${max_seq_len}
+
+    model:
+      name: hf_causal_lm
+      pretrained_model_name_or_path: mosaicml/mpt-7b-instruct
+      device: cpu
+      pretrained: true
+      use_auth_token: false
+  # -
+  #   model_name: falcon-40b
+  #   # Tokenizer
+  #   tokenizer:
+  #     name: tiiuae/falcon-40b
+  #     kwargs:
+  #       model_max_length: ${max_seq_len}
+
+  #   model:
+  #     name: hf_causal_lm
+  #     pretrained_model_name_or_path: tiiuae/falcon-40b
+  #     device: cpu
+  #     pretrained: true
+  #     use_auth_token: false
+  # -
+  #   model_name: mpt-30b
+  #   # Tokenizer
+  #   tokenizer:
+  #     name: EleutherAI/gpt-neox-20b
+  #     kwargs:
+  #       model_max_length: ${max_seq_len}
+
+  #   model:
+  #     name: hf_causal_lm
+  #     pretrained_model_name_or_path: mosaicml/mpt-30b
+  #     device: cpu
+  #     pretrained: true
+  #     use_auth_token: true
+  # -
+  #   model_name: falcon-40b-instruct
+  #   # Tokenizer
+  #   tokenizer:
+  #     name: tiiuae/falcon-40b-instruct
+  #     kwargs:
+  #       model_max_length: ${max_seq_len}
+
+  #   model:
+  #     name: hf_causal_lm
+  #     pretrained_model_name_or_path: tiiuae/falcon-40b-instruct
+  #     device: cpu
+  #     pretrained: true
+  #     use_auth_token: false
+  # -
+  #   model_name: llama-30b
+  #   # Tokenizer
+  #   tokenizer:
+  #     name: huggyllama/llama-30b
+  #     kwargs:
+  #       model_max_length: ${max_seq_len}
+
+  #   model:
+  #     name: hf_causal_lm
+  #     pretrained_model_name_or_path: huggyllama/llama-30b
+  #     device: cpu
+  #     pretrained: true
+  #     use_auth_token: false
+  -
+    model_name: mosaicml/mpt-7b
+    # Tokenizer
+    tokenizer:
+      name: EleutherAI/gpt-neox-20b
+      kwargs:
+        model_max_length: ${max_seq_len}
+
+    model:
+      name: hf_causal_lm
+      pretrained_model_name_or_path: mosaicml/mpt-7b
+      device: cpu
+      pretrained: true
+      use_auth_token: false
+  -
+    model_name: mosaicml/mpt-7b-chat
+    # Tokenizer
+    tokenizer:
+      name: EleutherAI/gpt-neox-20b
```

Comment:
This has a slightly different tokenizer compared to the Eleuther one with 2 extra tokens: mosaicml/mpt-7b-chat